robber/expect.py

```diff
@@ -60,9 +60,14 @@ def registered(cls, name):
         else:
             return True
 
+    @property
+    def not_to(self):
+        self.is_negated = True
+        return self
```

Comment:
if we have a `not` property then this is not needed. Since you could just write `.not.to`.